src/resources/books.md

```diff
@@ -3,21 +3,21 @@ title: Books about Dart
 description: Read all about it! Here's a collection of books about Dart.
 ---
 
-Here's a collection of books about the Dart language.
+This page covers a collection of books about the Dart language.
 Many [Flutter books](https://flutter.dev/docs/resources/books)
 also cover Dart.
-If you find another Dart book that might be helpful, please
+If you find another Dart book that might be helpful,
 [let us know.](https://github.com/dart-lang/site-www/issues)
 
 {{site.alert.warning}}
-  If you find a Dart book that isn't listed here,
-  make sure it was published or updated recently.
-  Older books lack coverage of Dart 2 topics such as
+  If you find a Dart book not listed on this page,
+  check for publication dates after June 2021.
+  Older books lack coverage of Dart 2 and 3 topics such as
   strong typing, null safety, FFI, the `dart` command-line utility,
-  and new developer tools. 
+  and new developer tools.
 {{site.alert.end}}
 
-## Dart books with null safety
+## Dart books
```

Comment:
Honestly you can drop this title now. It was only added for the differentiation between null-safe books and non-null safe ones. The whole page is about Dart books.

```suggestion
```